docs/source/services/timelock_service/service-management.rst

```diff
@@ -0,0 +1,70 @@
+.. _timelock-service-management:
+
+==================
+Service Management
+==================
+
+Namespace Changes
+=================
+
+.. danger::
+
+   Be careful when changing namespaces - doing this process incorrectly may result in **SEVERE DATA CORRUPTION**.
+   This guide also **ONLY** covers the steps for migrating a TimeLock client - AtlasDB clients, separately, must also
+   migrate the data in their key-value services to the new namespace.
```

Comment:
Link to docs?

Reply:
This link is on internal docs. I've added a line telling external and internal users steps they should do.